packages/api/src/store/db.ts

```diff
@@ -42,18 +41,6 @@ export interface PostgresParams {
 
 type Table<T> = BaseTable<WithID<T>>;
 
-type QueryHistogramLabels = {
-  query: string;
-  result: string;
-};
-
-const metricHistogram: Histogram<keyof QueryHistogramLabels> = new Histogram({
```

Comment:
we already have query insights enabled on gcp: https://console.cloud.google.com/sql/instances/livepeer-com/insights;duration=PT1H;sort_by=TOTAL_EXEC_TIME/executed?organizationId=251137649854&project=livepeer-firestore-production